requirements-optional.txt

```diff
@@ -1,14 +0,0 @@
-# For running tests and checking code quality using these modules.
-codecov==2.1.12
-flake8==6.0.0
-pytest-cov==3.0.0
-redis==4.4.0
-pre-commit==2.20.0
-
```

Comment:
why delete this file? :)

Reply:
During merged conflicts and due to bump that file got changed, added in commit by mistake.